pkg/operator2/secret.go

```diff
@@ -49,11 +70,12 @@ func newSessionSecretsJSON() []byte {
 		},
 		Secrets: []legacyconfigv1.SessionSecret{
 			{
-				Authentication: string(randomBits(sha256KeyLenBits)), // TODO these lengths are probably wrong
-				Encryption:     string(randomBits(aes256KeyLenBits)),
+				Authentication: string(randomBits(sha256KeyLenBits)), // 32 chars
```

Comment:
I think the issue here is that `randomBits` is just noise - it is not valid UTF8 so that cast to `string` is not valid.  I believe we have to base64 encode the value for it to be safe - and the length of the final base64 encoded string needs to be correct (so when osin casts it to `[]byte` it works out correctly).

Reply:
> the length of the final base64 encoded string needs to be correct

and that is Authentication: 64chars and Encryption: 32 chars?
Where is that requirement coming from?